docs/setup.md

```diff
@@ -14,7 +14,7 @@ In the future we'd like to provide it fully packaged inside a Docker container,
 
 Shipit provides you with a Rails template. To bootstrap your Shipit installation:
 
-1. If you don't have Rails installed, run this command: `gem install rails -v 7.0`
+1. If you don't have Rails installed, run this command: `gem install rails -v 7.1.1`
```

Comment:
Wherever it was previously just major.minor version, is it possible to stick to that? 7.1 instead of 7.1.1

Reply:
I am not sure if I am quite following this. Is the ask here to switch upgrade to 7.1 instead of 7.1.1?

I will DM you privately to vet this, Kate :)